### DISCUSSION TITLE
崔棉大师的Solidity8.0智能合约全面精通课程61P视频

### DISCUSSION THREAD

**Discussion by Fankouzu** [NonceGeek/Move-Camp#67] [Materials]
[油管链接](https://www.youtube.com/watch?v=XlJwBTIps_I&list=PLV16oVzL15MS-Zw8a3eEOADwbHhm8GrMp)
[B站链接](https://www.bilibili.com/medialist/play/286084162?from=space&business=space_collection&business_id=296410&desc=0)

课程源码以及更多实战案例网址: https://web3dao-cn.github.io/solidity-example/
关注我的推特: https://twitter.com/MasterCui

**Comment by leeduckgo** [comment]
Good!
there is an writer NFT waiting for send.
